bundler/tests/test.bats

```diff
@@ -0,0 +1,6 @@
+expected_version="$(echo $PKG_IDENT | cut -d/ -f3)"
+@test "bundler is expected version ${expected_version}" {
+  run hab pkg exec core/bundler bundle -v
+  [ $status -eq 0 ]
+  [[ "$output" =~ "${expected_version}" ]]
```

Comment:
Why are we doing a regex comparison here?

Also, be wary of the `[[` in bats. They are not required in this context either.

Reply:
I'll change this to a string comparison, but I used a regex so that our tests are not dependent on the rest of the version string. The version string presented might be reworded arbitrarily or localised.
I'll instead grep the last word from the string, so that we only depend on the version being at the end of the string.

Reply:
I'l all for being extremely strict on version checking. I want to know a package is `1.2.3` and not `1.2.3-alpha1`. As such, I use token separation to pull out the exact version string and do a direct string comparison. Theres no confusion or possibility of being shafted by an incorrect version that way.